lib/vagrant-service-manager/plugin.rb

```diff
@@ -1,3 +1,5 @@
+require_relative 'providers/docker'
```

Comment:
+1 service(s) - it really opens up the options for this plugin and makes the language so much better